envs/feedstock-patches/pyarrow/s390x_11686_11687.patch

```diff
@@ -0,0 +1,60 @@
+diff --git a/recipe/meta.yaml b/recipe/meta.yaml
+index 20fe865..6222870 100644
+--- a/recipe/meta.yaml
++++ b/recipe/meta.yaml
+@@ -11,6 +11,8 @@ package:
+ source:
+   url: https://github.com/apache/arrow/archive/refs/tags/apache-arrow-{{ version }}.tar.gz
+   sha256: {{ checksum }}
++  patches:
++    - s390x_arrow_11686_11687.patch
+ 
+ build:
+   number: 2
```

Comment:
Generally we update build numbers in the recipes. So I think we can set number to 3 here.

Reply:
ok